bin/copy-trilium.sh

```diff
@@ -18,16 +24,20 @@ mkdir -pv "$DIR"
 
 echo "Copying Trilium to build directory $DIR"
 
+mv build/* "$DIR"/
+rm -rf build
+
 for d in 'images' 'libraries' 'src' 'db'; do
     cp -r "$d" "$DIR"/
 done
-for f in 'package.json' 'package-lock.json' 'README.md' 'LICENSE' 'config-sample.ini' 'electron.js'; do
+for f in 'README.md' 'LICENSE' 'config-sample.ini'; do
     cp "$f" "$DIR"/
 done
-cp webpack-* "$DIR"/      # here warning because there is no 'webpack-*', but webpack.config.js only
+cp webpack-* "$DIR"/      # here warning because there is no 'webpack-*', but webpack.config.ts only
 
 # run in subshell (so we return to original dir)
-(cd $DIR && n exec 18.18.2 npm install --only=prod)
+# (cd $DIR && n exec 18.18.2 npm install --omit=dev && n exec 18.18.2 npm rebuild)
```

Comment:
Please remove the commented-out code. Personally I think it's OK to remove the dependency to `n` for managing the npm version.

Reply:
Removed. Nice catch!